assets/base/themes.json

```diff
@@ -20,7 +20,8 @@
         "buttonColor": "#FFFFFF",
         "mentionHighlightBg": "#fff2bb",
         "mentionHighlightLink": "#2f81b7",
-        "codeTheme": "github"
+        "codeTheme": "github",
+        "channelDropdownBg": "#000"
```

Comment:
We should check what the webapp does for the background colour of its dropdown. We shouldn't be adding fields here since the user's theme will actually be loaded as a JSON blob from their Preferences so it'll likely be missing that field. This `themes.json` will only be used once we add the ability to change themes from within the app